server/index.js

```diff
@@ -44,6 +44,95 @@ app.get('/api/products/:id', (req, res, next) => {
   }
 });
 
+app.get('/api/cart/', (req, res, next) => {
+  const sql = `
+  select "c"."cartItemId",
+         "c"."price",
+         "p"."productId",
+         "p"."image",
+         "p"."name",
+         "p"."shortDescription"
+    from "cartItems" as "c"
+    join "products"  as "p" using ("productId")
+   where "c"."cartId" = $1
+  `;
+  const value = [req.session.cartId];
+  db.query(sql, value)
+    .then(result => {
+      const data = result.rows;
+      res.status(201).json(data);
```

Comment:
Here a `200` response is sufficient. `201` means that something was created, but this was just a read operation.